code/src/java/pcgen/rules/persistence/CDOMControlLoader.java

```diff
@@ -126,7 +119,7 @@ private boolean parseSubLine(LoadContext context, String val, URI source)
 			return false;
 		}
 		String name = lineIdentifier.substring(colonLoc + 1);
-		if ((name == null) || name.isEmpty())
+		if (name.isEmpty())
```

Comment:
I'm getting confused by the inconsistent standards being applied. Why do we have a two part AND conditional in some cases, but in others we drop it to a single conditional?